scripts/mission_commander.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+
+import rospy
+import argparse
+import roslaunch
+import tf
+from std_msgs.msg import Int32
+from robosar_messages.srv import *
+from robosar_messages.msg import *
+from mtsp_ta import MtspCommander
+from greedy_ta import GreedyCommander
+
+
+class MissionCommander:
+
+    def __init__(self, args):
+        rospy.init_node('mission_commander', log_level=rospy.DEBUG)
+        rospy.logdebug("Initializing Mission Commander ...")
+        rospy.Subscriber('/mission_status', Int32, self.handle_commands)
```

Comment:
The topic name is very misleading, mission status indicates information but you are using that for commanding the task allocator, maybe rename it to something like

/system_mission_command?